src/controllers/JobRoleController.ts

```diff
@@ -1,8 +1,15 @@
 import express from "express";
 import { getJobRoles } from "../services/JobRoleService";
+import axios, { AxiosResponse } from "axios";
+
+axios.defaults.baseURL = process.env.API_URL || 'http://localhost:8080/';
```

Comment:
This can be set globally in the `index.ts` or the URL passed directly to the view. Please look into either of those approaches.

Reply:
![Screenshot 2024-07-31 at 08 24 56](https://github.com/user-attachments/assets/d03e4135-d796-419e-a6d0-916115f88e96)
Hi Stephanie, that is the base URL set globally in the index.ts file